### DISCUSSION TITLE
Terminal dosen't open

### DISCUSSION THREAD

**Discussion by theblackattacker** [lxqt/lxqt#2494] [Q&A]
##### Context

The qterminal dosen't work at all , when i try to open it , it dosen't open ever .. 

i can't use the system without the Terminal 

other app open corectoly , but only the Terminal doesent open , sometimes when i keep pressing on the icon , it open only for less then a second then immediatly disappear 

##### System Information
LXQt Desktop Toolbox - Technical Info<p>
  LXQt About Version:     0.16.0
  LXQt Version:           0.16.0
  Qt:                     5.15.2
  Build type:             Release
  System Configuration:   /etc/xdg
  Share Directory:        /usr/share/lxqt
  Translations:           /home/w/.local/share/lxqt/translations<br>
/usr/local/share/lxqt/translations<br>
/usr/share/lxqt/translations

User Directories
  Xdg Data Home:          /home/w/.local/share
  Xdg Config Home:        /home/w/.config
  Xdg Data Dirs:          /home/w/.local/share:/usr/local/share:/usr/share
  Xdg Cache Home:         /home/w/.cache
  Xdg Runtime Home:       /run/user/
  Xdg Autostart Dirs:     /etc/autostart<br>
/etc/xdg/autostart<br>
/usr/share/autostart
  Xdg Autostart Home:     /home/w/.config/autostart

**Comment by tsujan** [comment]
0.16.0 is outdated. The latest release is 1.4.0. Please upgrade.

**Comment by theblackattacker** [comment]
how can i upgrade without using Terminal ?

**Reply by Chiitoo** [reply]
Generally Linux will always provide you with a virtual terminal "behind your desktop", and you can access it via `CTRL-ALT-F1` (or anything up to F7; F1 might be used by your desktop environment, though often they use F7 instead).

**Reply by theblackattacker** [reply]
i already tried all of that , i couldnt open terminal till now ... the prob i use RDP , so i can't delet it and download the new version .. so what should i do ?

**Reply by tsujan** [reply]
Apparently you have Debian "oldstable" (bullseye). If so, it won't give you LXQt > 0.16 because "oldstable" is very old by definition. However, you could always install `xterm` to have a fallback terminal emulator (recommended) — unless RDP prevents that too.

A user of Debian bullseye could make better suggestions, although I don't think there are many users of bullseye+RDP.

**Reply by theblackattacker** [reply]
![2023-11-19_21052444](https://github.com/lxqt/lxqt/assets/56766902/605d27fe-ad81-422c-b1b5-75a67ea0c316)
 this appears when i keep press , shell no 1 then nothing to preview .. is there anyway that i can replace old Terminal Emulator to a new one ? , and can i install xtrem without terminal commands ?

**Reply by tsujan** [reply]
If you can right click inside that window, you'll see "Preferences..." on the menu. Click it to open QTerminal's preferences dialog, and make sure that "Terminal transparency" is zero in it.

If that didn't help, you could close all instances of QTerminal, rename the file `~/.config/qterminal.org/qterminal.ini` (`~` is your home), and launch QTerminal again to see if it makes a difference.

> can i install xtrem without terminal commands

As @Chiitoo said, you already have a virtual terminal. Read https://wiki.debian.org/AptCLI?highlight=%28%5CbCategorySoftware%5Cb%29 to know how to work with `apt` or `apt-get` (provided that RDP allows you to enter the root password).

EDIT:
As it's explained in Debian Wiki, you need to use `sudo` before `apt`, because `apt` requires root access.

**Comment by tsujan** [comment]
It depends on your distro.

**Comment by theblackattacker** [comment]
and how can i get it

**Comment by tsujan** [comment]
You may have Ubuntu, Debian, Arch, Fedora...; these are called "distros", and each one of them has its own way of package management.

I'm converting this page to a discussion.... There are users who could explain these things to you much better, especially those with the same distro.

**Reply by theblackattacker** [reply]
it's Debian
